src/Core/tests/DeviceTests/Handlers/Entry/EntryHandlerTests.Android.cs

```diff
@@ -15,6 +15,46 @@ namespace Microsoft.Maui.DeviceTests
 {
 	public partial class EntryHandlerTests
 	{
+		[Theory(DisplayName = "Validates Keyboard updates correctly using IsPassword")]
+		[InlineData(nameof(Keyboard.Text), false)]
+		[InlineData(nameof(Keyboard.Numeric), true)]
```

Comment:
I'm thinking we can just change this to a Fact if it's only testing the numeric keyboard. OR if you want this test to be more thorough, keep it as a Theory, test ALL keyboards generally, and make sure the expected keyboard is returned for each case so that the expected value is true for every test case.

```suggestion
		[Fact]
```

Reply:
Want to detect that works as expected with a Numeric keyboard but, the result is different with a Text keyboard.